app/routes/events/components/JoinEventForm.js

```diff
@@ -315,6 +307,20 @@ class JoinEventForm extends Component<Props> {
                       />
                     )}
                   </Form>
+                  {registration && showStripe && (
+                    <div style={{ width: '100%' }}>
+                      <div className={styles.joinHeader}>Betaling</div>
+                      <div className={styles.eventPrice}>
+                        Du skal betale {event.price / 100},-
```

Comment:
The decimal separator should be a comma, and `,-` should not be included if it is a decimal number. Use `kr` or `kroner` instead. 

https://www.sprakradet.no/svardatabase/sporsmal-og-svar/kronebelop-rekkjefolgje-komma-og-strek/